kiwi/volume_manager/lvm.py

```diff
@@ -167,7 +178,7 @@ def create_volumes(self, filesystem_name):
                 '--> volume %s with %s MB', volume.name, volume_mbsize
             )
             self._create_volume_no_zero(
-                [
+                self.lvm_tool_options, [
                     '-L', format(volume_mbsize), '-n',
```

Comment:
looks like a duplicate, _create_volume_no_zero already adds lvm_tool_options

Reply:
sorry, disregard. did not notice that _create_volume_no_zero got an addtional option.